bp_me/syn/Makefile

```diff
@@ -90,19 +90,19 @@ lint.me:
 	@echo "ME lint not supported"
 
 regress.me.sc: dirs.sc
-	$(MAKE) build.sc sim.sc TB=bp_cce NUM_INSTR_P=$(NUM_INSTR_P)
-	$(MAKE) build.sc sim.sc TB=bp_cce_uc
-	$(MAKE) build.sc sim.sc TB=bp_cce_fsm NUM_INSTR_P=$(NUM_INSTR_P)
-	$(MAKE) build.sc sim.sc TB=bp_cce_fsm_uc
+	$(MAKE) -j 1 build.sc sim.sc TB=bp_cce NUM_INSTR_P=$(NUM_INSTR_P)
```

Comment:
It basically just disables the job server.  If there's another way to force a job to run with only 1 thread (in the recipe), I'd prefer to do it that way.  I hesitate to just to 'make', because I'd still like users to be able to specify their MAKE program if they so choose.